src/variables.less

```diff
@@ -1,5 +1,5 @@
 @small-screen-width: 320px;
-@medium-screen-width: 510px;
+@medium-screen-width: 520px;
```

Comment:
The input lines wrapped too early if I left it at 510px. The 'pt' toggle takes up more space. I tested what it looks like at different breakpoints as you resize the screen and it works well